draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -697,7 +742,7 @@ Upon receiving an SD-JWT, a Holder or a Verifier needs to ensure that
 The Holder or the Verifier MUST perform the following (or equivalent) steps when receiving
 an SD-JWT:
 
-1. Separate the SD-JWT into the Issuer-signed JWT, the Disclosures (if any), and the Key Binding JWT (if present).
+1. Separate the SD-JWT into the Issuer-signed JWT and the Disclosures (if any).
```

Comment:
this is now a bit awkward as it kinda implies that, if it were an SD-JWT-KB, that it's already been separated into SD-JWT and KB parts. Which is different than prior to the PR.

Reply:
> it kinda implies that, if it were an SD-JWT-KB, that it's already been separated into SD-JWT and KB parts.

I am not sure it does. this section is being clear that it applies only to the verification of sd-jwt and not sd-jwt-kb. if this is still a concern, making sure verification by the verifier is probably a better section to claritfy this?

Reply:
Well it's a change from what was here before where this section was about general verification of the sd-jwt, which could have kb or not, and the process starts with splitting it up into the component parts. Maybe it doesn't matter though.

Reply:
Thanks, I made a minor suggestion to that wording but otherwise think it's good enough to allay my concern here.